src/playlist-maker.js

```diff
@@ -281,7 +295,25 @@ const factory = (player, initialList, initialIndex = 0) => {
     autoadvance(delay) {
       playlist.autoadvance_.delay = delay;
       autoadvance.setup(playlist.player_, delay);
+    },
+
+    /**
+     * Sets `repeat` option, which makes the "next" video of the last video
+     * is the first one.
```

Comment:
Suggestion: `"next" video of the last video is the first one.` -> `"next" video of the last video in the playlist be the first video in the playlist`